notebooks/4.0-ldb-neighborhood-applied-with-kmeans.ipynb

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
<p>You are repeating some imports, I suggest you to create a cell (the first code one) just with imports!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>